production/helm/loki/values.yaml

```diff
@@ -14,8 +14,12 @@ affinity: {}
 ## Deployment annotations
 annotations: {}
 
+# enable tracing for debug, need install jaeger and specify right jaeger_agent_host
+jaegerAgentHost:
```

Comment:
I like the tracing: part bring it back :)

Reply:
I think comment is enough.